tests/integration_tests/test_arming.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import time
+
+import pytest
+from PyQt5 import QtWidgets
+
+import main
+from DebugConnectionFactory import DebugConnectionFactory
+
+
+def test_arm_signal(qtbot, capsys):
+    factory = DebugConnectionFactory()
+    connection = factory.construct()
+    main_window = main.MainApp(connection)
+
+    time.sleep(1)
+    main_window.sendCommand("arm")
+    time.sleep(1)
+    captured = str(capsys.readouterr())
+
+    assert captured.find("b'r' sent to DebugConnection") != -1
```

Comment:
I added an if statement that checks if the assert is about to fail. If so, it will wait 15 seconds and get the console again. I feel like that is a more-than-generous delay, especially considering how simplistic DebugConnection's implementation of the send method is. If the ground station takes more than 16 seconds total to confirm that a command was sent, even when disregarding any code related to the radio (since we're using DebugConnection), then I feel like we'd have bigger problems than a failing integration test. Let me know if you disagree though.